### PR TITLE
Run pull-kubernetes-e2e-ec2 on all PR(s) but do not report

### PR DESCRIPTION
**Pull request by dims**
Let's take `pull-kubernetes-e2e-ec2` for a test drive and see what happens :)

### CODE REVIEW

**Comment by dims** [comment]
/assign @tzneal @upodroid 

cc @kubernetes/sig-testing-leads @kubernetes/sig-testing-pr-reviews

File: config/jobs/kubernetes/sig-testing/ec2-e2e.yaml [hunk 1]
```diff
@@ -11,8 +11,9 @@ presubmits:
       preset-e2e-containerd-ec2: "true"
       preset-dind-enabled: "true"
     path_alias: k8s.io/kubernetes
-    always_run: false
+    always_run: true
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-testing/ec2-e2e.yaml:14]: can we please add a description or a better name? "GCE" is understood to be the shell scripts, with newer tools we've included something clearer referencing the bootstrapping tool "kops" or "kind". or if this is node e2es and not cluster e2es we can add "node" to the name.

Comment (dims) [reply config/jobs/kubernetes/sig-testing/ec2-e2e.yaml:14]: This uses `kubetest2-ec2`[1]  to stand up the cluster. I'll try to make it clearer in the description.

[1] https://github.com/kubernetes-sigs/provider-aws-test-infra/tree/main/kubetest2-ec2#readme

Comment (dims) [reply config/jobs/kubernetes/sig-testing/ec2-e2e.yaml:14]: this is not a node e2e job. This is the equivalent for `pull-kubernetes-e2e-gce` for the record.

**Comment by dims** [comment]
/skip 

`pull-test-infra-gubernator` seems unrelated

File: config/jobs/kubernetes/sig-testing/ec2-e2e.yaml (hunk 1 above)
Comment (BenTheElder) [reply config/jobs/kubernetes/sig-testing/ec2-e2e.yaml:15]: I see. I still think the naming is vague and contributors etc probably won't know what this means.

**Pull request merged**